Makefile

```diff
@@ -332,7 +347,10 @@ ifeq ($(BUNDLE_DEBUG_LIBS),1)
 endif
 
 	# Copy in all .jl sources as well
-	cp -R -L $(build_datarootdir)/julia $(DESTDIR)$(datarootdir)/
+	mkdir -p $(DESTDIR)$(datarootdir)/julia/{base,test}
```

Comment:
Some systems don't use `bash` as their make shell; we can't use the `{base,test}` expansion here.  :(